.buildkite/pipeline.yml

```diff
@@ -35,8 +46,7 @@ steps:
      MONEYBAG: "unencrypted:edsk3GjD83F7oj2LrnRGYQer99Fj69U2QLyjWGiJ4UoBZNQwS38J4v"
    commands: &run-nettest
      - nix build -L -f. nettest
-     - mkdir -p test/resources/
-     - nix-build -A tezos-contract -o test/resources/stablecoin.tz
+     - bash -c "$FETCH_CONTRACT"
```

Comment:
Isn't it better to call `/usr/bin/env bash` here and above? It probably shouldn't affect anything, just a matter of style.

Reply:
I think that `eval "$FETCH_CONTRACT"` better suits here, dunno, why I chose `bash -c` here